packages/main/test/pages/Toast.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta charset="utf-8">
+	<title>ui5-toast</title>
+
+	<script src="../../webcomponentsjs/webcomponents-loader.js"></script>
+	<script src="../../resources/bundle.esm.js" type="module"></script>
+	<script nomodule src="../../resources/bundle.es5.js"></script>
+	<script>delete Document.prototype.adoptedStyleSheets;</script>
+	<script src="modules/Toast.js"></script>
+
+	<style>
+		ui5-button, h2 {
+			margin: 1rem;
+		}
+	</style>
+</head>
+
+<body onload="onload()">
+
+	<h2>Toast ( ui5-toast )</h2>
+
+	<!--TopStart-->
+	<ui5-button class="wcBtnOpenToastTS">Show TopStart Toast</ui5-button>
+	<ui5-toast class="wcToastTS" placement="TopStart">TopStart</ui5-toast>
+
+	<br>
+
+	<!--TopCenter-->
+	<ui5-button class="wcBtnOpenToastTC">Show TopCenter Toast</ui5-button>
```

Comment:
In the html file you have wcBtnOpenToastTC, but in the JS file you look for wcBtnShowToastTC,
it applies for all toasts.